{"createdAt": "2023-11-12T17:57:11.279413+00:00", "files": [{"name": "OT2_None_None_2_13_PythonSyntaxError.py", "role": "main"}, {"name": "opentrons_ot3_96_tiprack_1000ul_rss.json", "role": "labware"}, {"name": "sample_labware.json", "role": "labware"}, {"name": "opentrons_ot3_96_tiprack_50ul_rss.json", "role": "labware"}, {"name": "opentrons_ot3_96_tiprack_200ul_rss.json", "role": "labware"}, {"name": "cpx_4_tuberack_100ul.json", "role": "labware"}], "config": {"protocolType": "python", "apiVersion": [2, 13]}, "metadata": {"protocolName": "bad import", "author": "Opentrons Engineering <engineering@opentrons.com>", "source": "Software Testing Team", "description": "import superspecialmagic", "apiLevel": "2.13"}, "robotType": "OT-2 Standard", "commands": [{"id": "c6583c50-240e-4820-ba0a-7ad80b1f5b57", "createdAt": "2023-11-12T17:57:11.270515+00:00", "commandType": "home", "key": "50c7ae73a4e3f7129874f39dfb514803", "status": "succeeded", "params": {}, "result": {}, "startedAt": "2023-11-12T17:57:11.270751+00:00", "completedAt": "2023-11-12T17:57:11.270872+00:00"}], "labware": [{"id": "fixedTrash", "loadName": "opentrons_1_trash_1100ml_fixed", "definitionUri": "opentrons/opentrons_1_trash_1100ml_fixed/1", "location": {"slotName": "12"}}], "pipettes": [], "modules": [], "liquids": [], "errors": [{"id": "953c7bf5-19ed-44e1-b36b-0f7ab1bf2204", "createdAt": "2023-11-12T17:57:11.275258+00:00", "errorCode": "4000", "errorType": "UnexpectedProtocolError", "detail": "No module named 'superspecialmagic'", "errorInfo": {}, "wrappedErrors": [{"id": "953c7bf5-19ed-44e1-b36b-0f7ab1bf2204", "createdAt": "2023-11-12T17:57:11.275258+00:00", "errorCode": "4000", "errorType": "PythonException", "detail": "ModuleNotFoundError: No module named 'superspecialmagic'", "errorInfo": {"args": "(\"No module named 'superspecialmagic'\",)", "msg": "No module named 'superspecialmagic'", "name": "superspecialmagic", "path": "None", "traceback": "  File \"/usr/local/lib/python3.10/site-packages/opentrons/protocol_runner/task_queue.py\", line 72, in _run\n    await self._run_func()\n\n  File \"/usr/local/lib/python3.10/site-packages/opentrons/protocol_runner/legacy_wrappers.py\", line 173, in execute\n    await to_thread.run_sync(run_protocol, protocol, context)\n\n  File \"/usr/local/lib/python3.10/site-packages/anyio/to_thread.py\", line 28, in run_sync\n    return await get_asynclib().run_sync_in_worker_thread(func, *args, cancellable=cancellable,\n\n  File \"/usr/local/lib/python3.10/site-packages/anyio/_backends/_asyncio.py\", line 802, in run_sync_in_worker_thread\n    return await future\n\n  File \"/usr/local/lib/python3.10/site-packages/anyio/_backends/_asyncio.py\", line 740, in run\n    result = func(*args)\n\n  File \"/usr/local/lib/python3.10/site-packages/opentrons/protocols/execution/execute.py\", line 27, in run_protocol\n    run_python(protocol, context)\n\n  File \"/usr/local/lib/python3.10/site-packages/opentrons/protocols/execution/execute_python.py\", line 46, in run_python\n    exec(proto.contents, new_globs)\n\n  File \"OT2_None_None_2_13_PythonSyntaxError.py\", line 4, in <module>\n", "class": "ModuleNotFoundError"}, "wrappedErrors": []}]}]}